cmd/experimental/kjobctl/pkg/cmd/delete/delete_slurm.go

```diff
@@ -0,0 +1,218 @@
+	"sigs.k8s.io/kueue/cmd/experimental/kjobctl/pkg/constants"
+)
+
+const (
+	slurmExample = `  # Delete Slurm 
+  kjobctl delete slurm my-application-profile-slurm-k2wzd`
+)
+
+type SlurmOptions struct {
+	PrintFlags *genericclioptions.PrintFlags
+
+	JobNames  []string
+	Namespace string
+
+	CascadeStrategy metav1.DeletionPropagation
+	DryRunStrategy  util.DryRunStrategy
+
+	Clientset k8s.Interface
+
+	PrintObj printers.ResourcePrinterFunc
+
+	genericiooptions.IOStreams
+}
+
+func NewSlurmOptions(streams genericiooptions.IOStreams) *SlurmOptions {
+	return &SlurmOptions{
+		PrintFlags: genericclioptions.NewPrintFlags("deleted").WithTypeSetter(scheme.Scheme),
+		IOStreams:  streams,
+	}
+}
+
+func NewSlurmCmd(clientGetter util.ClientGetter, streams genericiooptions.IOStreams) *cobra.Command {
+	o := NewSlurmOptions(streams)
+
+	cmd := &cobra.Command{
+		Use:                   "slurm NAME [--cascade STRATEGY] [--dry-run STRATEGY]",
+		DisableFlagsInUseLine: true,
+		Short:                 "Delete Slurm",
+		Example:               slurmExample,
+		Args:                  cobra.MinimumNArgs(1),
+		ValidArgsFunction:     completion.JobNameFunc(clientGetter, v1alpha1.SlurmMode),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			cmd.SilenceUsage = true
+
+			err := o.Complete(clientGetter, cmd, args)
+			if err != nil {
+				return err
+			}
+
+			return o.Run(cmd.Context())
+		},
+	}
+
+	addCascadingFlag(cmd)
+	util.AddDryRunFlag(cmd)
+
+	o.PrintFlags.AddFlags(cmd)
+
+	return cmd
+}
+
+func (o *SlurmOptions) Complete(clientGetter util.ClientGetter, cmd *cobra.Command, args []string) error {
+	o.JobNames = args
+
+	var err error
+
+	o.Namespace, _, err = clientGetter.ToRawKubeConfigLoader().Namespace()
+	if err != nil {
+		return err
+	}
+
+	o.DryRunStrategy, err = util.GetDryRunStrategy(cmd)
+	if err != nil {
+		return err
+	}
+
+	err = util.PrintFlagsWithDryRunStrategy(o.PrintFlags, o.DryRunStrategy)
+	if err != nil {
+		return err
+	}
+
+	o.CascadeStrategy, err = getCascadingStrategy(cmd)
+	if err != nil {
+		return err
+	}
+
+	printer, err := o.PrintFlags.ToPrinter()
+	if err != nil {
+		return err
+	}
+
+	o.PrintObj = printer.PrintObj
+
+	o.Clientset, err = clientGetter.K8sClientset()
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func (o *SlurmOptions) Run(ctx context.Context) error {
+	for _, jobName := range o.JobNames {
+		if err := o.deleteJob(ctx, jobName); err != nil {
+			return err
+		}
+		if err := o.deleteConfigMap(ctx, jobName); err != nil {
+			return err
+		}
+	}
+
+	return nil
+}
+
+func (o *SlurmOptions) deleteJob(ctx context.Context, jobName string) error {
+	job, err := o.Clientset.BatchV1().Jobs(o.Namespace).Get(ctx, jobName, metav1.GetOptions{})
+	if client.IgnoreNotFound(err) != nil {
+		return err
+	}
+	if err != nil {
+		fmt.Fprintln(o.ErrOut, err)
+		return nil
+	}
+	if _, ok := job.Labels[constants.ProfileLabel]; !ok {
+		fmt.Fprintf(o.ErrOut, "jobs.batch \"%s\" not created via kjob\n", job.Name)
```

Comment:
```suggestion
		fmt.Fprintf(o.ErrOut, "jobs.batch \"%s\" not owned by kjobctl\n", job.Name)
```
Nit: Maybe like this?

Reply:
We already discussed it [here](https://github.com/kubernetes-sigs/kueue/pull/2839#discussion_r1722277918). I would like it to be as in other places. WDYT?

Reply:
I see. Then it's fine, let's keep it as is :)